### DISCUSSION TITLE
Select class item by value?

### DISCUSSION THREAD

**Discussion by creyes-cricutor** [HC200ok/vue3-easy-data-table#117] [Ideas]
IT should be possible select class item by value of this.

For example if value < 50 -> classA
else classB

**Comment by HC200ok** [comment]
@creyes-cricutor You can use the item slot feature to customize body item like this:
<template #item-value="item">
     <div :class="[item.value < 50 ? 'classA' : 'classB']">{{item.value}}</div>
</template>

**Reply by creyes-cricutor** [reply]
Thank you for the anwser @HC200ok 
Finally I have done this:

const bodyRowClassNameFunction: BodyRowClassNameFunction = (
  item: Item,
  index: number
): string => {
  **ComStatusValue** = item.ComStatus;
  return "";
};

const bodyItemClassNameFunction: BodyItemClassNameFunction = (
  column: string,
  index: number
): string => {
  if (column === "ComStatus") {
    if (**ComStatusValue** === "A") return "active-status";
    else return "pf-status";
  } else {
    return "";
  }
};

**Reply by nclemeur** [reply]
I think that approach of defining a class using the BodyRowClassNameFunction is the right one, but it would be a lot more convenient to have the possibility to define the function in the headers instead of a global function... The BodyRowClassNameFunction  could still be used for any column not parts of the headers.
